src/CMakeLists.txt

```diff
@@ -49,6 +49,9 @@ if(NOT CMAKE_BUILD_TYPE STREQUAL Debug)
       CMAKE_C_FLAGS_${build_type_upper})
     string(REGEX REPLACE "(^| )[/-]D *NDEBUG($| )" " " "${flags}" "${${flags}}")
   endforeach()
+else()
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CMAKE_CXX_FLAGS_DEBUG} -O0 -g" )
```

Comment:
@tchaikov 
If you look at ./do_freebsd.sh you see that I'm trying to get  those options set, but they just don't make it into the CMake. So that is why I keep trying. :)
If you have any suggestion why the commandline options do not get thru I'll fix that, and we can really remove them.